bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/service/impl/BahmniPatientServiceImplTest.java

```diff
@@ -1,5 +1,7 @@
 package org.bahmni.module.bahmnicore.service.impl;
 
+import org.bahmni.module.bahmnicore.contract.patient.PatientSearchParameters;
```

Comment:
presumably this change isn't necessary since there's no changes to the class itself